src/components/ChatHeader/ChatHeader.jsx

```diff
@@ -0,0 +1,12 @@
+import { ChatHeaderStyled, HeaderContent } from './ChatHeaderStyled';
+
+const ChatHeader = ({ countryData }) => (
+  <ChatHeaderStyled>
+    <HeaderContent>
```

Comment:
Why do you need these 2 components "ChatHeaderStyled", "HeaderContent"? I think one of them is redundant.

Reply:
Done